rmu_gazebo_simulator/scripts/referee_system/simple_competition_1v1.py

```
@@ -1,3 +1,17 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 #!/usr/bin/python3
```

Ruff (0.14.2)

Shebang should be at the beginning of the file

(EXE005)